packages/ts/lit-form/src/BinderNode.ts

```diff
@@ -259,7 +359,7 @@ export class BinderNode<M extends AbstractModel = AbstractModel> extends EventTa
 
     this.initializeValue();
 
-    const key = this.model[_key];
+    const key = this.model[$key];
```

Comment:
I was confused seeing a `$key` here. Probably, it requires a small description in a doc comment